tests/ecp5/fsm.ys

```diff
@@ -0,0 +1,14 @@
+read_verilog fsm.v
+hierarchy -top top
+proc
+flatten
+#ERROR: Found 4 unproven $equiv cells in 'equiv_status -assert'.
+#equiv_opt -assert -map +/ecp5/cells_sim.v synth_ecp5 # equivalency check
```

Comment:
TODO

Reply:
Not sure why this should fail.... Firstly @SergeyDegtyar, does the result pass simulation?

Reply:
Yes, this test passes simulation in Iverilog. Checked it manually.

Reply:
This test passes in #1421 now..!?